wpeverest/user-registration
# UR-400 Fix- Disable Default WordPress Login Screen not working on custom login page

### All Submissions:






### Changes proposed in this Pull Request:

Previously, when a custom login page is created using our myaccount or login page shortcode then Disable Default WordPress Login Screen on the custom login page is not working. This PR will fix these issues.
Please refer to this Link to replicate:
https://prnt.sc/xUrBZKkvyzB9

### How to test the changes in this Pull Request:

1. Create a page using both Gutenberg block and classic editor with content in it. like paragraphs and images.
2. then make checked Disable Default WordPress Login Screen and choose the login page as the custom login page.
3. and verify whether Disable Default WordPress Login Screen on the custom login page is working or not in all possible ways.

### Types of changes:




### Other information:




### Changelog entry

> Fix- Disable Default WordPress Login Screen not working on the custom login page.
